src/maintenance/remove_outdated_keys.py

```diff
@@ -0,0 +1,73 @@
+from collections import namedtuple
+from smtplib import SMTP
+
+import delphi.operations.secrets as secrets
+import mysql.connector
+from delphi.epidata.server._config import API_KEY_REGISTRATION_FORM_LINK_LOCAL
+
+API_USER_RECORD = namedtuple("APIUser", ("api_key", "email", "date_diff"))
+
+SMTP_HOST = "relay.andrew.cmu.edu"
+SMTP_PORT = 25
+
+EMAIL_SUBJECT = "Your API Key was deleted."
+EMAIL_FROM = "noreply@andrew.cmu.edu"
+EMAIL_MESSAGE = f"""Hi! \n Your API Key was removed due to inactivity.
+To get new one, please use registration form ({API_KEY_REGISTRATION_FORM_LINK_LOCAL}) or contact us."""
+
+
+def get_outdated_keys(cur):
+    cur.execute(
+        """
+            SELECT
+                diff.api_key,
+                diff.email,
+                diff.date_diff
+            FROM (
+                SELECT
+                    api_key,
+                    email,
+                    created,
+                    last_time_used,
+                    ABS(TIMESTAMPDIFF(MONTH, last_time_used, created)) as date_diff
+                FROM api_user
+            ) diff
+            WHERE diff.date_diff >= 5;
```

Comment:
i thought we were going to send a warning at 5 months, but then delete after 6 months?

Reply:
Yeah, check that in `main` function:
```
for item in outdated_keys_list:
        if item.date_diff == 5:
            send_notification(item.email)
        else:
            remove_outdated_key(cur, item.api_key)
```

Reply:
ah, ok!